content/docs/reference/spec/migration/buildpack-api-0.6-0.7.md

```diff
@@ -0,0 +1,36 @@
++++
+title="Buildpack API 0.6 -> 0.7"
++++
+
+<!--more-->
+
+This guide is most relevant to buildpack authors.
+
+See the [spec release](https://github.com/buildpacks/spec/releases/tag/buildpack%2Fv0.7) for buildpack API 0.7 for the full list of changes and further details.
+
+### New standardized SBOM format
+
+Buildpacks may write Software Bill of Materials (SBOM) files describing build- or run-time dependencies. These files must use the `application/vnd.cyclonedx+json`, `application/spdx+json`, or `application/vnd.syft+json` media types (a buildpack may output SBOM files in multiple formats). Files may be written to the following locations:
+
+* `<layers>/<layer>.sbom.<ext>` - for describing dependencies associated with a layer
+  * Example (launch layer): libraries that were included in an output compiled binary (e.g., `go` libraries in a `go` binary)
+  * Example (build layer): tools like a package manager
+* `<layers>/launch.sbom.<ext>` - for describing run-time dependencies not associated with a layer
+  * Example: dependencies installed in the `/workspace` directory
+* `<layers>/build.sbom.<ext>` - for describing build-time dependencies not associated with a layer
+  * Example: build time configuration
```

Comment:
We could probably have a whole docs section on SBOM stuff. This seemed like the minimum information needed for now.

Reply:
Looked plenty sufficient to me.